### PR TITLE
[14.0][Mig] osi_payment_method

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

File: osi_payment_method/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019 Open Source Integrators
```
Comment (patrickrwilson) [inline osi_payment_method/__init__.py]: Update copyright date on all files it's referenced.

File: osi_payment_method/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to Contacts
+* Create or select a partner
+* Go to the "Sales & Purchase" tab
```
Comment (patrickrwilson) [inline osi_payment_method/README.rst]: It looks like this field is now in the 'Accounting' tab so this should be updated.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Tested and works, just need readme updated and copyright dates updated.

**Review by patrickrwilson — APPROVED** [review]
Tested and works. LGTM

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**